docs/network-flow-visibility.md

```diff
@@ -754,6 +755,15 @@ Mouse out or click on the background will bring all the traffic back.
 
 <img src="https://downloads.antrea.io/static/08032022/flow-visibility-np-3.png" width="900" alt="Network-Policy Flows Dashboard">
 
+#### Network Topology Dashboard
+
+Network Topology Dashboard visualizes Pod-to-Pod and Pod-to-Service traffic.
+Pods labels are visually grouped by Node. Each arrow points from the source to
+the destination, and is labelled by the cumulative amount of data transmitted
+in the selected time range.
+
+<img src="https://downloads.antrea.io/static/03012023/flow-visibility-network-topology-0.png" width="900" alt="Network Topology Dashboard service dependency graph">
```

Comment:
could you adjust the width to a smaller value? Currently it's too big, I need to scroll down to see the entire picture.